### ISSUE TITLE
Using `six` before it is installed from `setup.py`

### ISSUE THREAD

**Issue by selitvin** [uber/petastorm#69]
`setup.py` would `import petastorm` to get the `__version__`
However, `petastorm/__init__.py` has `import six`, which would fail.

To reproduce:
```
$ virtualenv .env --no-site-packages && source .env/bin/activate && pip install -e .
New python executable in .env/bin/python
Installing setuptools, pip...done.
Obtaining file:///home/yevgeni/petastorm
  Running setup.py (path:/home/yevgeni/petastorm/setup.py) egg_info for package from file:///home/yevgeni/petastorm
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/home/yevgeni/petastorm/setup.py", line 18, in <module>
        from petastorm import __version__
      File "petastorm/__init__.py", line 16, in <module>
        import six
    ImportError: No module named six
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/home/yevgeni/petastorm/setup.py", line 18, in <module>

    from petastorm import __version__

  File "petastorm/__init__.py", line 16, in <module>

    import six

ImportError: No module named six

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /home/yevgeni/petastorm
Storing debug log for failure in /home/yevgeni/.pip/pip.log
```

**Comment by rgruener** [comment]
We should probably move out base classes from `petastorm/__init__.py`

**Comment by forbearer** [comment]
Resolved: #99
